src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -301,11 +303,27 @@ internal void Authenticate()
 
             State = UploadState.Uploading;
 
-            IReachHttpResponse result;
+            IReachHttpResponse result = null;
 
             try
             {
-                result = await Task.Run(() => this.Send(workspace, workspaceProperties));
+                var customizerExists = await this.CheckCustomizer(workspaceProperties.Name);
+
+                if (customizerExists) // there is a customizer with specified name
+                {
+                    MessageBoxResult decision =
+                        MessageBox.Show("A customizer by this name already exists, do you want to overwrite it?",
+                            "Confirmation",
```

Comment:
As stated in the above code, we'll need to make sure that all of this is extracted to resource files. 

"User facing strings, if any, are extracted into *.resx files"

@lillismith Would it be acceptable to show a message box here?